evm/v0.5/src/helpers.ts

```diff
@@ -650,10 +901,21 @@ export function findEventIn(
 }
 
 /**
- * Calculate six months from the current date in seconds
+ * Compute function signature hash for a function on an ethersjs Contract instance
+ * @param contract ethersjs contract
+ * @param functionName The function name
  */
-export function sixMonthsFromNow(): ethers.utils.BigNumber {
-  return ethers.utils.bigNumberify(
-    Math.round(Date.now() / 1000.0) + 6 * 30 * 24 * 60 * 60,
-  )
+export function getFunctionSignature(
+  contract: ethers.Contract | ethers.ContractFactory,
+  functionName: string,
+): string {
+  return contract.interface.functions[functionName].sighash
+}
+
+/**
+ * Compute function signature hash for an arbitrary string function
+ * @param functionName The function name ex: "funName(uint256,bytes32)"
+ */
+export function encodeFunctionSignature(functionName: string) {
```

Comment:
I don't think we need this function, all of our sighash requirements are met by ethers. For every time we need a function signature, we can do either `myContract.functions.myFunc.sighash` or `myContractFactory.interface.functions.myFunc.sighash`. I would prefer the former since its more strictly typed.